fs/stat_linux.go

```diff
@@ -22,5 +22,5 @@ func StatMtime(st *syscall.Stat_t) syscall.Timespec {
 
 // StatATimeAsTime returns st.Atim as a time.Time
 func StatATimeAsTime(st *syscall.Stat_t) time.Time {
-	return time.Unix(st.Atim.Sec, st.Atim.Nsec)
+	return time.Unix(int64(st.Atim.Sec), int64(st.Atim.Nsec)) // nolint: unconvert
```

Comment:
Include a comment here why these conversions are necessary. Otherwise, they may get removed.

Reply:
@stevvooe I'm willing to address this in a follow up PR, but if you don't want it, I'm okay opening a carry-PR, just let me know.

Reply:
Sounds good.